.github/composite/exp1/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Run experiment 1
+description: "Run Gradle Experiment 1"
+
+inputs:
+  repositoryUrl:
+    description: "Project repository URL"
+    required: true
+  branch:
+    description: "Git Branch"
+    required: true
+  task:
+    description: "Gradle task"
+    required: true
+  projectDir:
+    description: "Project directory"
+    required: false
+    default: .
+  gradleEnterpriseUrl:
+    description: "Gradle Enterprise URL"
+    required: true
+outputs:
+  buildScanId1:
+    description: "First build scan identifier"
+    value: ${{ steps.run.outputs.buildScanId1 }}
```

Comment:
Is it just the build scan id or the entire build scan URL? (the latter would make more sense to me) If so, I would rename the field to buildScaFirstBuild and buildScanSecondBuild to be aligned with the naming we use in the output of the underlying validation scripts.

Reply:
This is the build scan identifier as it used down the line to query the REST API